full-service/src/db/models.rs

```diff
@@ -68,24 +85,38 @@ pub struct NewAccount<'a> {
     pub name: &'a str,
 }
 
+/// A received transaction output entity that belongs to a an Account in this
+/// wallet. Also maybe a transaction output that hasn't been sent yet?
 #[derive(Clone, Serialize, Identifiable, Queryable, PartialEq, Debug)]
 #[primary_key(id)]
 pub struct Txo {
+    /// Primary key
     pub id: i32,
+    /// ???
```

Comment:
Again, what's the story with this and the primary key 'id'?

Reply:
This one might have more merit as a separate field than the account_id_hex - the reason for this is that you may get a Txo in different contexts, and you want to look up if you already have it in your DB, so you use the reproducible txo_id_hex, derived from the TxOut itself.